test/isImplementedCache.test.ts

```diff
@@ -14,22 +14,28 @@ describe('ImplementedCache', () => {
         isImplementedCache.add(event, true);
 
         expect(isImplementedCache.get(event)).toBeDefined();
-        expect(isImplementedCache.get(event)?.isImplemented).toBeTrue();
+        expect(isImplementedCache.get(event)?.isImplemented).toBe(true);
       }
     );
     it('should overwrite a specific event that already exists in cache', () => {
       isImplementedCache.add(event, true);
-      isImplementedCache.add(event, false);
+      isImplementedCache.add(event, true);
 
       expect(isImplementedCache.get(event)).toBeDefined();
-      expect(isImplementedCache.get(event)?.isImplemented).toBeFalse();
+      expect(isImplementedCache.get(event)?.isImplemented).toBe(false);
```

Comment:
How can that be false, if you set true ?

Reply:
resolved